src/particle/pdgid/functions.py

```diff
@@ -165,9 +168,10 @@ def is_baryon(pdgid):
         return True
     if _extra_bits(pdgid) > 0:
         return False
-    if _fundamental_id(pdgid) > 0 and _fundamental_id(pdgid) <= 100:
+    if _fundamental_id(pdgid) in range(1, 101):
```

Comment:
IIRC, in Python 3, this is optimized to basically do the check above. :) (In Python 2, it generates the 100 numbers and then checks each on, since range is not a generator)

Reply:
Oh, then I should revert back. Good point.

Reply:
I think it's okay to not worry about optimizing for Python 2. But let me make a suggestion (will in a minute)

Reply:
OK, I will wait for your green light / suggestion. Otherwise I'm done here.